packages/generator/src/service/readme.ts

```diff
@@ -32,6 +32,11 @@ function businessDocumentationLine(service: VdmServiceMetadata): string[] {
     : [];
 }
 
+// TODO 1728 move to a new package for reduce code duplication.
+/**
+ * Build the helpful links section of the readme file.
+ * @returns The content of the section.
+ */
```

Comment:
Perhaps you can use the content of the generic readme I introduced in this PR: https://github.com/SAP/cloud-sdk-js/pull/974
 also for the readme section.

Reply:
We can also mention this in the ticket 1728 to do that.

Reply:
created a follow up